contracts/registry.go

```diff
@@ -49,12 +48,16 @@ func init() {
 
 // GetRegisteredAddress returns the address on the registry for a given id
 func GetRegisteredAddress(vmRunner vm.EVMRunner, registryId common.Hash) (common.Address, error) {
+	if vmRunner == nil {
```

Comment:
Can this be nil?  Is it also due to the tracer test?  If so, same comment on it as https://github.com/celo-org/celo-blockchain/pull/1522/files#r632775742

Reply:
removed